packages/tmux/tmux.pacscript

```diff
@@ -0,0 +1,23 @@
+name="tmux"
+version="3.2a"
+url="https://github.com/tmux/tmux/releases/download/${version}/tmux-${version}.tar.gz"
+build_depends="gcc make pkg-config bison"
+depends="libncurses-dev libevent-dev"
+breaks="tmux-git"
+replace="tmux"
+description="A terminal multiplexer"
+hash="551553a4f82beaa8dadc9256800bcc284d7c000081e47aa6ecbb6ff36eacd05f"
+optdepends=("libutempter-dev: Record user sessions to utmp and wtmp files [add to depends array and rebuild to enable]")
+maintainer="Harshwardhan Mehrotra <abrakadabra18262@protonmail.com>"
+prepare() {
+          ./autogen.sh
+}
+build() {
+        ./configure
```

Comment:
```suggestion
        # Check if `libutempter-dev` ( optdepends ) is installed
        if dpkg-query -W libutempter-dev > /dev/null 2>&1; then
                ./configure --enable-utempter
        else
                ./configure
        fi
```

Suppose the user enables `libutempter`. You need to add the `--enable-utempter` flag here to enable it. My suggestion figures out if the user has installed the lib, and proceeds accordingly.

Reply:
Fixed in https://github.com/pacstall/pacstall-programs/pull/361/commits/615b145c09a3c6bd509325f5cba8e977f1e09e50